test/unit/util/CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ add_executable(util_assert_failure assert.F90)
 set_target_properties(util_assert_failure PROPERTIES LINKER_LANGUAGE Fortran)
 add_std_test_script(util_assert_failure assert.sh)
 
+set_tests_properties(util_assert_failure PROPERTIES WILL_FAIL TRUE)
+
```

Comment:
```suggestion
```
The script actually catches the failures and makes sure the correct failure code is sent and then exits successfully

Reply:
I see that now in the unit testing; it fails on the command line in a way that appears to be a true failure, as opposed to the other `failure` cases.  I'll make this change, thanks!